packages/logger/src/index.ts

```diff
@@ -473,7 +473,7 @@ export class QuietLogger {
   private _getLogSetting(): LogSetting {
     if (this._canTrace()) {
       return LogSetting.TRACE
-    } else if (debug.enabled(this.name)) {
+    } else if (debug.enabled('*') || debug.enabled(this.name)) {
```

Comment:
These changes allow setting `DEBUG=*` or `DEBUG=*:trace` to quickly set log levels globally.  I got tired of rewriting the debug string with trace everytime I wanted to test.